pkg/http/server_test.go

```diff
@@ -58,413 +52,6 @@ func newServer() server {
 	}
 }
 
-func TestAllowedAPISpec(t *testing.T) {
```

Comment:
Reviewers: these tests were moved to the endpoints_test  file (with new tests added)